docker-compose.yml

```diff
@@ -21,13 +21,19 @@ version: '3.5'
 
 x-ubuntu-volumes:
   &ubuntu-volumes
-  - .:/arrow:delegated
-  - ${ARROW_DOCKER_CACHE_DIR:-./docker_cache}/ubuntu:/build:delegated
+  - .:/arrow:ro  # ensures that docker won't contaminate the host directory
```

Comment:
This brought up new issues.

Reply:
Hm, can you rsync the files into container space and build there?

Reply:
Yes, but I don't like that solution, but probably have to stick with that.